src/App.tsx

```diff
@@ -1,14 +1,69 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+function filterTodos(todos: Todo[], todosActivityFilter: string, todosQuery: string) {
+  let resultTodos = [...todos];
+  const query = todosQuery.toLowerCase();
+
+  switch (todosActivityFilter) {
```

Comment:
Create a type for `todosActivityFilter`. You could use enum but I would personally go for an union type ;)